index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
         </div>
       </section>
     </main>
+    <section class="project-section">
+      <div class="project-container flex column">
+        <div class="project-card flex column">
+          <img src="./images/Snapshoot Portfolio.svg" alt="project card image">
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card flex column">
+          <img src="./images/Snapshoot Portfolio (1).svg" alt="project card image">
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
```

Comment:
- [ ] Kindly change the text `color` of the technologies to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205633529-607c705c-cbe9-4f89-a74e-a511a693b1b7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205633566-8db38eb0-3b52-4f40-aff6-cf7b0c3a7af0.png)